apps/condo/domains/billing/schema/BillingReceipt.test.js

```diff
@@ -469,7 +469,9 @@ describe('BillingReceipt', () => {
 
                             expect(receipts).toHaveLength(0)
                         })
-                        test('If serviceConsumer context is wrong', async () => {
+                        // billingIntegrationContext is deprecated as it is not take part in business logic
+                        // also there is no legal way for resident to change billing or acquiring context in service consumer
+                        test.skip('If serviceConsumer context is wrong', async () => {
```

Comment:
I think its better to delete these tests

Reply:
Done